[011] Отображение зона под землёй

degonismi opened the pull request:
Добавил коллайдеры для островов, сделал чтобы рейкаст от пушек реагировал на коллайдеры

degonismi committed:
001

Добавил коллайдеры для островов

degonismi committed:
002

Рейкаст от  пушек реагирует на коллайдеры островов

degonismi committed:
Merge pull request #1 from degonismi/011_Display_UnderGround_Zone

011 display under ground zone

HolyMonkey commented on ThousandDevils/Assets/Scripts/AffectedArea.cs:
```diff
@@ -31,15 +31,23 @@ private void Update()
+                if (hit.collider.GetComponent<ShipHealth>())
+                {
+                    areaColor = Color.green;
+                    lineRendererEndPosition = hit.point;
```
Поправь код. У тебя есть одинаковые код как в if так и в else

HolyMonkey commented on ThousandDevils/ProjectSettings/ProjectSettings.asset:
```diff
@@ -106,11 +106,6 @@ PlayerSettings:
   xboxOneDisableEsram: 0
```
Изменения этого файла коммитить не надо. Откати их чтобы не отображалось разницы

degonismi committed:
Revert "Merge pull request #1 from degonismi/011_Display_UnderGround_Zone"

This reverts commit 4681406b479968030c43b91b6a36ee8331b3cb28, reversing
changes made to 4d692cd4f5e055637064deab5f5179eeeb75aa36.

degonismi committed:
Revert "Revert "Merge pull request #1 from degonismi/011_Display_UnderGround_Zone""

This reverts commit 724cbd73a71ddf506af0457f804b7a36b3076163.

degonismi committed:
Revert "Revert "Revert "Merge pull request #1 from degonismi/011_Display_UnderGround_Zone"""

This reverts commit d7808f61b85119d0fdce723e4c3732eb2bae4c6e.

degonismi committed:
Revert "Revert "Merge pull request #1 from degonismi/011_Display_UnderGround_Zone""

This reverts commit 724cbd73a71ddf506af0457f804b7a36b3076163.

degonismi committed:
Revert "002"

This reverts commit afab267e3675df7b798dd1a5fd3af6376c0dae5c.

degonismi committed:
Revert "001"

This reverts commit 7d7dc0468dfa6a3f5433b776af2f74b06faab71d.

degonismi committed:
Revert "Revert "001""

This reverts commit 2fa2e9074ec1df8560e3f6879f265e57a17acda9.

degonismi committed:
Revert "001"

This reverts commit 7d7dc0468dfa6a3f5433b776af2f74b06faab71d.

degonismi committed:
011

Задал цвет дефолтный цвет

HolyMonkey commented on ThousandDevils/Assets/Scripts/AffectedArea.cs:
```diff
@@ -31,20 +31,26 @@ private void Update()
+                if (hit.collider.GetComponent<ShipHealth>())
+                {
+                    areaColor = Color.green;
+                    lineRendererEndPosition = hit.point;
```
if(hit.collider.GetComponent<ShipHealth>())
    areaColor = Color.green;
else
    areaColor = Color.gray;

lineRendererEndPosition = hit.point;

degonismi committed:
011-Remake

задание  дефолтного цвета оставил, чтобы не дублировать в if (wasHitted) else...
